### PR TITLE
Create tag names files.

### PR DESCRIPTION
**Pull request by suecarmol**
## Description
Create `en` and `qqq` files for tags so TranslateWiki can pick them up.

## Rationale
[//]: # (Why is this change required? What problem does it solve?)
This is the first step that will make tag names translatable.

## Phabricator Ticket
[//]: # (Link to the Phabricator ticket)
[T281013](https://phabricator.wikimedia.org/T281013)

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Minor change (fix a typo, add a translation tag, add section to README, etc.)

### CODE REVIEW

File: locale/en/tag_names.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+			"Susana Cárdenas Molinar",
+			"Jason Sherman",
+			"Sam Walton"
+		],
+		"locale": "en",
+		"message-documentation": "qqq"
+	},
+    "language_translation": "language",
+    "multidisciplinary_translation": "multidisciplinary",
+    "physical-science_translation": "physical-science",
```
Comment (Samwalton9) [inline locale/en/tag_names.json:13]: I think these tags had no spaces/capital letters in the past for technical reasons. I think they could all be properly capitalised and have hyphens removed in favour of spaces (so in this case **Physical science**).

File: locale/en/tag_names.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "music_translation": "music",
+    "art_translation": "art",
+    "social_translation": "social",
+    "sciences_translation": "sciences",
```
Comment (jsnshrmn) [inline locale/en/tag_names.json:17]: I think that these two tags exist by mistake, and that the one resource that has them should probably be using the `social-sciences` tag.

**Review by jsnshrmn — COMMENTED** [review]
seems like our message keys should be different? instead of `art_translation` maybe we should have `art_tag` or `tag_art`?

File: locale/en/tag_names.json (hunk 1 above)
Comment (suecarmol) [reply locale/en/tag_names.json]: Done!

File: locale/en/tag_names.json (hunk 2 above)
Comment (suecarmol) [reply locale/en/tag_names.json]: Those message keys make more sense. Have removed both tags.

**Review by jsnshrmn — APPROVED** [review]
👍

**Pull request merged**